default.nix

```diff
@@ -251,8 +251,7 @@ in emacs.overrideAttrs (esuper:
           local -a wrapArgs=(
               --set NIX_DOOM_EMACS_BINARY $1
               --set-default DOOMDIR ${doomDir}
-              --set-default __DEBUG_doom_emacs_DIR ${doom-emacs}
-              --set-default __DEBUG_doomLocal_DIR ${doomLocal}
```

Comment:
Did a grep in the `doomemacs` repository. Those variables don't seem to be used anymore.

Reply:
And they probably never were, they're just for debugging (convenient `M-x getenv`)

Reply:
I see. So do you want it to return?

Reply:
Mhm

Reply:
Mhm

Reply:
Done.